src/__test__/Button.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import Button from '../components/Button';
+
+describe('<Button />', () => {
+  test('should render a button primary', () => {
+    const btn = shallow(<Button classes='btn--primary' value='Click' onClick={jest.fn()}/>);
+
+    expect(toJson(btn)).toMatchSnapshot();
+  });
+
+  test('should be get prop value', () => {
+    const btn = shallow(<Button classes='btn--primary' value='Jest' onClick={jest.fn()} />);
```

Comment:
It is ok. But you can also just check that prop was passed correctly from your snapshots. Just don't forget to check snapshots by yourself when creating them (otherwise they are useless).